cmake/modules/RootCPack.cmake

```diff
@@ -29,12 +29,26 @@ string(REGEX REPLACE "^([0-9]+).*$" "\\1" CXX_MAJOR ${CMAKE_CXX_COMPILER_VERSION
 string(REGEX REPLACE "^([0-9]+)\\.([0-9]+).*$" "\\2" CXX_MINOR ${CMAKE_CXX_COMPILER_VERSION})
 
 #---Resource Files-----------------------------------------------------------------------------------
-configure_file(README.md README.md COPYONLY)
 configure_file(LICENSE LICENSE.txt COPYONLY)
 configure_file(LGPL2_1.txt LGPL2_1.txt COPYONLY)
-set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_BINARY_DIR}/README.md")
 set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_BINARY_DIR}/LICENSE.txt")
-set(CPACK_RESOURCE_FILE_README "${CMAKE_BINARY_DIR}/README.md")
+if (APPLE)
+  find_program(CONVERTER textutil)
+  if (NOT CONVERTER)
+    message(FATAL_ERROR "textutil executable not found")
+  endif()
+  execute_process(COMMAND ${CONVERTER} -convert html README.md)
+else()
+  find_program(CONVERTER markdown)
```

Comment:
Isn't `pandoc` more common than `markdown` these days?

Reply:
@Axel-Naumann Neither `pandoc` nor `markdown` seem to come preinstalled :(
Do you have any suggestion on what we could use to convert a `md` file to another format like `html` or `rtf`?
If such a tool doesn't come preinstalled, maybe we should just rename it to txt instead?